frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -594,45 +594,33 @@ describe("scenarios > collection_defaults", () => {
     });
 
     describe("bulk actions", () => {
-      it("should be possible to apply bulk selection to items (metabase#14705)", () => {
-        cy.visit("/collection/root");
-        selectItemUsingCheckbox("Orders");
-        cy.findByText("1 item selected").should("be.visible");
-
-        // Select all
-        cy.icon("dash").click();
-        cy.icon("dash").should("not.exist");
-        cy.findByText("4 items selected");
-
-        // Deselect all
-        cy.findByTestId("bulk-action-bar").within(() => {
-          cy.icon("check").click();
+      describe("selection", () => {
```

Comment:
This is already a good indicator that this should be a separate file, but we can follow up on that.
```
bulk actions
  selection
  move
  archive
```